hooks/push.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+/**
+ * GitHub webhook to send emails using push event
+ */
+
+error_reporting(E_ALL);
+
+define('PMAHOOKS', True);
+
+require_once(__DIR__.'/lib/github.php');
+
+if ( !is_file(__DIR__.'/vendor/autoload.php')) {
+    fail("Run composer install");
+    exit;
+}
+
+//Load Composer's autoloader
+require_once(__DIR__.'/vendor/autoload.php');
+
+github_verify_post();
+
+
+$inputData = json_decode(file_get_contents('php://input'));
+
+if (json_last_error() !== 0) {
+    fail("JSON body was not well formed.");
+    exit;
+}
+
+
+$data = gihub_webhook_push($inputData);
+
+
+$mail = new PHPMailer(true);
+try {
+
+    $mail->SMTPDebug  = 0;
+    $mail->isSMTP();
+    $mail->Host       = SMTP_HOST;
+    $mail->SMTPAuth   = true;
+    $mail->Username   = SMTP_USERNAME;
+    $mail->Password   = SMTP_PASSWORD;
+    $mail->SMTPSecure = SMTP_MODE;
+    $mail->Port       = SMTP_PORT;
```

Comment:
Can you please also support sendmail here? That would make it much easier to configure on existing infrastructure - sendmail is already working there, while we don't have remote SMTP server set up...

Reply:
Yes, I will make the `SMTP_` optional.
phpmailer supports calling `mail()`